Description: <nothing>
 Nothing
 .
 usermanage-client (1.2.0-1) unstable; urgency=low
 .
   * Add head image editor
Author: levi <91he@sina.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ usermanage-client-1.2.0/config.log
@@ -0,0 +1,181 @@
+This file contains any messages produced by compilers while
+running configure, to aid debugging if configure makes a mistake.
+
+It was created by user-manage-client configure 1.1.0, which was
+generated by GNU Autoconf 2.69.  Invocation command line was
+
+  $ ./configure --build=i686-linux-gnu --prefix=/usr --includedir=${prefix}/include --mandir=${prefix}/share/man --infodir=${prefix}/share/info --sysconfdir=/etc --localstatedir=/var --libdir=${prefix}/lib/i386-linux-gnu --libexecdir=${prefix}/lib/i386-linux-gnu --disable-maintainer-mode --disable-dependency-tracking
+
+## --------- ##
+## Platform. ##
+## --------- ##
+
+hostname = levi-TEST
+uname -m = i686
+uname -r = 3.8.13.13-cdos
+uname -s = Linux
+uname -v = #1 SMP Mon Nov 17 14:31:23 CST 2014
+
+/usr/bin/uname -p = unknown
+/bin/uname -X     = unknown
+
+/bin/arch              = unknown
+/usr/bin/arch -k       = unknown
+/usr/convex/getsysinfo = unknown
+/usr/bin/hostinfo      = unknown
+/bin/machine           = unknown
+/usr/bin/oslevel       = unknown
+/bin/universe          = unknown
+
+PATH: /usr/local/sbin
+PATH: /usr/local/bin
+PATH: /usr/sbin
+PATH: /usr/bin
+PATH: /sbin
+PATH: /bin
+
+
+## ----------- ##
+## Core tests. ##
+## ----------- ##
+
+configure:2250: checking for a BSD-compatible install
+configure:2318: result: /usr/bin/install -c
+configure:2329: checking whether build environment is sane
+configure:2384: result: yes
+
+## ---------------- ##
+## Cache variables. ##
+## ---------------- ##
+
+ac_cv_env_CC_set=
+ac_cv_env_CC_value=
+ac_cv_env_CFLAGS_set=set
+ac_cv_env_CFLAGS_value='-g -O2 -fstack-protector --param=ssp-buffer-size=4 -Wformat -Werror=format-security'
+ac_cv_env_CPPFLAGS_set=set
+ac_cv_env_CPPFLAGS_value=-D_FORTIFY_SOURCE=2
+ac_cv_env_CPP_set=
+ac_cv_env_CPP_value=
+ac_cv_env_LDFLAGS_set=set
+ac_cv_env_LDFLAGS_value='-Wl,-Bsymbolic-functions -Wl,-z,relro'
+ac_cv_env_LIBS_set=
+ac_cv_env_LIBS_value=
+ac_cv_env_build_alias_set=set
+ac_cv_env_build_alias_value=i686-linux-gnu
+ac_cv_env_host_alias_set=
+ac_cv_env_host_alias_value=
+ac_cv_env_target_alias_set=
+ac_cv_env_target_alias_value=
+ac_cv_path_install='/usr/bin/install -c'
+
+## ----------------- ##
+## Output variables. ##
+## ----------------- ##
+
+ACLOCAL=''
+AMDEPBACKSLASH=''
+AMDEP_FALSE=''
+AMDEP_TRUE=''
+AMTAR=''
+AM_BACKSLASH=''
+AM_DEFAULT_V=''
+AM_DEFAULT_VERBOSITY=''
+AM_V=''
+AUTOCONF=''
+AUTOHEADER=''
+AUTOMAKE=''
+AWK=''
+CC=''
+CCDEPMODE=''
+CFLAGS='-g -O2 -fstack-protector --param=ssp-buffer-size=4 -Wformat -Werror=format-security'
+CPP=''
+CPPFLAGS='-D_FORTIFY_SOURCE=2'
+CYGPATH_W=''
+DEFS=''
+DEPDIR=''
+ECHO_C=''
+ECHO_N='-n'
+ECHO_T=''
+EGREP=''
+EXEEXT=''
+GREP=''
+GTK_CFLAGS=''
+GTK_LIBS=''
+INSTALL_DATA='${INSTALL} -m 644'
+INSTALL_PROGRAM='${INSTALL}'
+INSTALL_SCRIPT='${INSTALL}'
+INSTALL_STRIP_PROGRAM=''
+LDFLAGS='-Wl,-Bsymbolic-functions -Wl,-z,relro'
+LIBOBJS=''
+LIBS=''
+LTLIBOBJS=''
+MAKEINFO=''
+MKDIR_P=''
+OBJEXT=''
+PACKAGE=''
+PACKAGE_BUGREPORT='NULL'
+PACKAGE_NAME='user-manage-client'
+PACKAGE_STRING='user-manage-client 1.1.0'
+PACKAGE_TARNAME='user-manage-client'
+PACKAGE_URL=''
+PACKAGE_VERSION='1.1.0'
+PATH_SEPARATOR=':'
+PKG_CONFIG=''
+SET_MAKE=''
+SHELL='/bin/bash'
+STRIP=''
+VERSION=''
+ac_ct_CC=''
+am__EXEEXT_FALSE=''
+am__EXEEXT_TRUE=''
+am__fastdepCC_FALSE=''
+am__fastdepCC_TRUE=''
+am__include=''
+am__isrc=''
+am__leading_dot=''
+am__nodep=''
+am__quote=''
+am__tar=''
+am__untar=''
+bindir='${exec_prefix}/bin'
+build_alias='i686-linux-gnu'
+datadir='${datarootdir}'
+datarootdir='${prefix}/share'
+docdir='${datarootdir}/doc/${PACKAGE_TARNAME}'
+dvidir='${docdir}'
+exec_prefix='NONE'
+host_alias=''
+htmldir='${docdir}'
+includedir='${prefix}/include'
+infodir='${prefix}/share/info'
+install_sh=''
+libdir='${prefix}/lib/i386-linux-gnu'
+libexecdir='${prefix}/lib/i386-linux-gnu'
+localedir='${datarootdir}/locale'
+localstatedir='/var'
+mandir='${prefix}/share/man'
+mkdir_p=''
+oldincludedir='/usr/include'
+pdfdir='${docdir}'
+prefix='/usr'
+program_transform_name='s,x,x,'
+psdir='${docdir}'
+sbindir='${exec_prefix}/sbin'
+sharedstatedir='${prefix}/com'
+sysconfdir='/etc'
+target_alias=''
+
+## ----------- ##
+## confdefs.h. ##
+## ----------- ##
+
+/* confdefs.h */
+#define PACKAGE_NAME "user-manage-client"
+#define PACKAGE_TARNAME "user-manage-client"
+#define PACKAGE_VERSION "1.1.0"
+#define PACKAGE_STRING "user-manage-client 1.1.0"
+#define PACKAGE_BUGREPORT "NULL"
+#define PACKAGE_URL ""
+
+configure: caught signal 2
+configure: exit 1
